msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerMSALController.java

```diff
@@ -188,6 +190,9 @@ private static BrokerRequest getBrokerRequestForInteractive(MSALAcquireTokenOper
         request.setName(parameters.getLoginHint());
         request.setRedirect(parameters.getRedirectUri());
         request.setScope(StringUtil.join(' ', parameters.getScopes()));
+        String extraQP = BrokerRequest.getJsonStringForExtraQueryParams(parameters.getExtraQueryStringParameters());
```

Comment:
See comment [here](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/313#discussion_r237582637) regarding `BrokerRequest` -- might be better to let the `QueryParamsAdapter` handle this